pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -3801,10 +3801,10 @@ func TestResolvePipelineRunTask_WithMatrix(t *testing.T) {
 		name: "task with matrix - whole array results",
 		pt:   pts[2],
 		want: &ResolvedPipelineTask{
-			TaskRunNames: nil,
+			TaskRunNames: []string{"pipelinerun-pipelinetask-with-whole-array-results"},
```

Comment:
The current behavior will return a definition of TaskSpec, facilitating subsequent verification.